src/backend/database.mo

```diff
@@ -164,6 +165,16 @@ module {
       });
     };
 
+    public func findProjectsByTerm(term: Text) : [Project] {
```

Comment:
Please make sure this function only projects that are not in the "submitted" state. Otherwise, this will return any project that has been submitted even before we have reviewed.